docs/writing-docs/autodocs.md

```diff
@@ -83,7 +83,7 @@ Going over the code snippet in more detail. When Storybook starts up, it will ov
 
 #### With MDX
 
-Nevertheless, not all projects are built by adopting a single convention. There may be some situations where you want to use MDX to generate the documentation template. In that case, you can customize the template by adding the [`Meta`](../api/doc-block-meta.md) Doc Block and supplying the `isTemplate` property to tell Storybook to treat this file as a documentation template. For example:
+You can also use MDX to generate the documentation template. This is useful in non-React projects where JSX-handling is not configured. Normally, when you create an MDX file in your project, it is treated as normal documentation. To indicate that an MDX file is a documentation template, supply an the `isTemplate` property to its [`Meta`](../api/doc-block-meta.md) Doc Block. For example:
```

Comment:
One small tweak here, if you could remove the `an` after supply, turning this into:
"You can also use MDX to generate the documentation template. This is useful in non-React projects where JSX handling is not configured. Normally, when you create an MDX file in your project, it is treated as normal documentation. To indicate that an MDX file is a documentation template, supply the `isTemplate` property to its [`Meta`](../api/doc-block-meta.md) Doc Block. For example:"

Reply:
Good catch! Fixed.